front/src/scenes/master.scene.tsx

```diff
@@ -2,7 +2,14 @@ import React from 'react';
 import { CreateSessionContainer } from 'pods/create-session';
 import { CenteredLayout } from 'layout';
 import { MasterContainer } from 'pods/master';
+import { Header, Footer } from 'common-app/components';
 
 export const MasterScene: React.FC = () => {
-  return <MasterContainer />;
+  return (
+    <CenteredLayout>
+      <Header />
+      <MasterContainer />
+      <Footer />
+    </CenteredLayout>
```

Comment:
I think we should create a layout called: AppLayout

That already enclose the header and the footer

This App Layout will be used by the other two scenes

Reply:
resolved in new commit